dev/windows/setup.ps1

```diff
@@ -0,0 +1,36 @@
+$phpDir = Read-Host "Enter the path of your PHP installation (e.g. 'C:\Program Files\php', 'D:\php', 'C:\wamp\bin\php')"
```

Comment:
This could be changed to a [script parameter](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_parameters?view=powershell-7.5) so that the script doesn't require input once invoked (e.g. `dev\windows\setup.ps1 -PhpDir "C:\wamp\bin\php"`)

Then, the flag could be made optional, and infer the location of `php` if it is present in `$env:PATH`.